whotracksme/data/assets/trackerdb.sql

```diff
@@ -3498,6 +3536,26 @@ INSERT INTO "trackers" VALUES('sape.ru','Sape',6,NULL,'sape',NULL,NULL);
 INSERT INTO "trackers" VALUES('schibsted','Schibsted Media Group',8,'http://www.schibsted.com/','schibsted_asa',NULL,NULL);
 INSERT INTO "trackers" VALUES('shopifycdn.com','Shopify CDN',9,'https://www.shopify.com/','shopify','3064',NULL);
 INSERT INTO "trackers" VALUES('shopifycloud.com','Shopify Cloud',2,'https://www.shopify.com/','shopify',NULL,NULL);
+INSERT INTO "trackers" VALUES('orange_france','Orange France',8,'https://www.orange.fr/','orange_france',NULL,NULL);
+INSERT INTO "trackers" VALUES('dmcdn.net','Dailymotion',0,'https://www.dailymotion.com/','dailymotion',NULL,NULL);
+INSERT INTO "trackers" VALUES('le_monde.fr','Le Monde.fr',8,'http://www.lemonde.fr/','le_monde.fr',NULL,NULL);
+INSERT INTO "trackers" VALUES('accengage','Accengage',4,'https://www.accengage.com/','accengage',NULL,NULL);
+INSERT INTO "trackers" VALUES('polar.me','Polar',4,'https://polar.me/','polar','2111',NULL);
+INSERT INTO "trackers" VALUES('nekudo.com','Nekudo',2,'https://nekudo.com/','nekudo',NULL,NULL);
+INSERT INTO "trackers" VALUES('wikia-services.com','Wikia Services',8,' http://www.wikia.com/fandom','wikia',NULL,NULL);
+INSERT INTO "trackers" VALUES('baidu_static','Baidu Static',8,'https://www.baidu.com/','baidu',NULL,NULL);
+INSERT INTO "trackers" VALUES('allegro','Allegro',8,'allegroimg.com','allegro.pl',NULL,NULL);
+INSERT INTO "trackers" VALUES('allegrostatic.com','Allegro',8,'https://allegro.pl/','allegro.pl',NULL,NULL);
+INSERT INTO "trackers" VALUES('pushwoosh.com','Pushwoosh',2,'https://www.pushwoosh.com/','pushwoosh',NULL,NULL);
+INSERT INTO "trackers" VALUES('instart_logic','Instart Logic',4,'https://www.instartlogic.com/','instart_logic_inc',NULL,NULL);
+INSERT INTO "trackers" VALUES('the_guardian','The Guardian',8,'https://www.theguardian.com/','the_guardian',NULL,NULL);
+INSERT INTO "trackers" VALUES('onfocus.io','OnFocus',4,'http://onfocus.io/','onfocus','3071',NULL);
+INSERT INTO "trackers" VALUES('fastpic.ru','FastPic',10,'http://fastpic.ru/','fastpic',NULL,NULL);
+INSERT INTO "trackers" VALUES('vk.com','Vk.com',7,'https://vk.com/','vk.com',NULL,NULL);
+INSERT INTO "trackers" VALUES('os.tc','OneSignal',5,'https://os.tc/','onesignal','2498',NULL);
+INSERT INTO "trackers" VALUES('monero_miner','Monero Miner',8,'http://devappgrant.space/','monero',NULL,NULL);
```

Comment:
Is this service actually run by the Monero company?

Reply:
I think so... http://devappgrant.space/faq/ they link to getmonero.org and mymonero.com. I didn't find other info on the topic. If it is not reliable I'll delete this entry.

Reply:
It seems very unofficial to me. I think we should remove the company entry and leave this as without a company.